mso/resource_mso_schema_template_bd.go

```diff
@@ -141,10 +141,11 @@ func resourceMSOTemplateBD() *schema.Resource {
 				Computed: true,
 			},
 			"dhcp_policy": &schema.Schema{
-				Type:        schema.TypeMap,
-				Optional:    true,
-				Description: "Configure dhcp policy in versions before NDO 3.2",
-				Computed:    true,
+				Type:          schema.TypeMap,
+				Optional:      true,
+				Description:   "Configure dhcp policy in versions before NDO 3.2",
+				Computed:      true,
```

Comment:
I'm not sure of the behaviour of this resource for this attribute when it is computed for NDO < 3.2  and NDO >= 3.2 due to the version check. Hence this question:
Does this attribute get populated at all when a single dhcp_policy is created using the other attribute dhcp_policies?

Reply:
dhcp_policies - Only for NDO >= 3.7 and dhcp_policy is only for MSO. So we should use proper attributes to configure the DHCP Policy based on the NDO/MSO version.

Also, the resource will set the dhcp_policy and dhcp_policies values based on the MSO/NDO version.